### ISSUE TITLE
Disable Resize, just move the crop-area

### ISSUE THREAD

**Issue by hamidyfine** [alexk111/ngImgCrop#144]
Hi,
I want to set size for crop-area and disable it for user, and user just move the crop-area.
How can I achieve this?
Thanks

**Comment by thierryskoda** [comment]
+1

**Comment by rickerd** [comment]
Check our repo.
https://github.com/CrackerakiUA/ngImgCropFullExtended

**Comment by shamimafridi** [comment]
@rickerd  Which attribute of ui-cropper we should set to archive this?

**Comment by rickerd** [comment]
I don't know what you want to achieve but this lib. is super old.
Since quite a while the new ui-cropper is maintained somewhere else.
https://github.com/CrackerakiUA/ui-cropper

All other instances of ngCropper can be removed. We have the most up-to-date plugin with the most and best features.

**Comment by shamimafridi** [comment]
Yes i am using ui-cropper and now i posted new issue on ui-cropper. actually i want just to disable the resize option of the cropper.
